common/artifacts/000_wc_artifacts.txt

```diff
@@ -785,8 +777,7 @@ gauntlets_of_trollbanes = {
 	slot = wrist
 }
 
-### Cloak ###
-
+### ceremonial_torso ###
```

Comment:
when was cloak slot deleted? I like that :)

Reply:
Long time ago, but category name is still here.